.github/workflows/test-e2e.yml

```
@@ -37,3 +37,10 @@ jobs:
       - name: End-to-End Tests
         run: npm run test.end-to-end -- --ci
         shell: bash
+
+      - name: Git status check
+        # here we check that there are no changed / new files from our test
+        # run. we use `git status`, grep out the build zip (downloaded above),
+        # and check if there are more than 0 lines in the output.
+        run: if [[ $(git status --short | grep -v stencil-core-build.zip | wc -l) -ne 0 ]]; then exit 1; fi
```

No action required - we can shorten this with grep's `-c` flag just a smidge:
```suggestion
        run: if [[ $(git status --short | grep -c -v stencil-core-build.zip) -ne 0 ]]; then exit 1; fi
```

but that's purely stylistic according to https://www.shellcheck.net/wiki/SC2126.  Arguably (IMO) the way its written now is more readable, but I'm not super familiar with all of grep's flags.  Approving, LMK your thoughts - either way I think we'll be able to merge this tomorrow

---

ohh managed not to know about `-c` for `grep`! yeah I think possibly using `wc` might make it more obvious what's going on, but on the other hand to figure out the `grep` flags requires like one minute w/ the terminal and `man` so....I think it's probably fine to use `-c -v`